source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/action/DropdownElementRenderer.java

```diff
@@ -82,14 +86,24 @@ public Button render(RenderedAdaptiveCard renderedCard, Context context, Fragmen
         dropDownItem.setMinWidth(Util.dpToPixels(context, 100));
         dropDownItem.setMinimumWidth(0);
 
-        if (baseActionElement.GetIconUrl() != null && !baseActionElement.GetIconUrl().isEmpty())
+        if (!iconUrl.isEmpty())
         {
-            dropDownItem.post(() ->
+            IconPlacement iconPlacement = hostConfig.GetActions().getIconPlacement();
+            if (!renderArgs.getAllowAboveTitleIconPlacement())
             {
-                Drawable[] drawables = button.getCompoundDrawablesRelative();
-                dropDownItem.setCompoundDrawablePadding(button.getCompoundDrawablePadding());
-                dropDownItem.setCompoundDrawablesRelativeWithIntrinsicBounds(drawables[0], drawables[1], drawables[2], drawables[3]);
-            });
+                iconPlacement = IconPlacement.LeftOfTitle;
+            }
+
+            ActionElementRendererIconImageLoaderAsync imageLoader = new ActionElementRendererIconImageLoaderAsync(
+                renderedCard,
+                dropDownItem,
+                hostConfig.GetImageBaseUrl(),
+                iconPlacement,
+                hostConfig.GetActions().getIconSize(),
+                hostConfig.GetSpacing().getDefaultSpacing(),
+                context
+            );
+            imageLoader.executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR, iconUrl);
```

Comment:
I believe this is identical to code in ActionElementRenderer. Consider moving it into a helper within ActionElementRenderer and calling it, instead of duplicating.

Reply:
moved to helper class.